<launch>
  <arg name="reconfig_motion_manager" default="1"/>
  <group if="$(arg reconfig_motion_manager)">
  <node pkg="explore_lite" type="explore" respawn="false" name="explore" output="screen">
    <rosparam file="$(find omniveyor_mobility)/param/explore_params.yaml" command="load" />
    <remap from="move_base/goal" to="motion_manager/move_base/goal"/>
    <remap from="move_base/cancel" to="motion_manager/move_base/cancel"/>
    <remap from="move_base/status" to="motion_manager/move_base/status"/>
    <remap from="move_base/feedback" to="motion_manager/move_base/feedback"/>
    <remap from="move_base/result" to="motion_manager/move_base/result"/>
  </node>
  </group>
  <group unless="$(arg reconfig_motion_manager)">
  <node pkg="explore_lite" type="explore" respawn="false" name="explore" output="screen">
    <rosparam file="$(find omniveyor_mobility)/param/explore_params.yaml" command="load" />
  </node>
  </group>
</launch>
